public/icons/search-icon.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
```

Comment:
Just wonder if this file could even be more optimized. Please try [this](https://jakearchibald.github.io/svgomg/)